tests/test_widerface_format.py

```diff
@@ -113,6 +113,26 @@ def test_can_save_dataset_with_no_subsets(self):
 
             compare_datasets(self, source_dataset, parsed_dataset)
 
+    @mark_requirement(Requirements.DATUM_GENERAL_REQ)
+    def test_dataset_with_save_dataset_meta_file(self):
+        source_dataset = Dataset.from_iterable([
+            DatasetItem(id='a/b/1', image=np.ones((8, 8, 3)),
+                subset='train', annotations=[
+                    Bbox(0, 2, 4, 2, label=2),
+                    Bbox(0, 1, 2, 3, label=1, attributes={
+                        'blur': '2', 'expression': '0', 'illumination': '0',
+                        'occluded': '0', 'pose': '2', 'invalid': '0'}),
+                ]
+            ),
+        ], categories=['face', 'label_0', 'label_1'])
+
+        with TestDir() as test_dir:
+            WiderFaceConverter.convert(source_dataset, test_dir, save_images=True,
+                save_dataset_meta=True)
+            parsed_dataset = Dataset.import_from(test_dir, 'wider_face')
+
+            compare_datasets(self, source_dataset, parsed_dataset)
```

Comment:
I think, it would be reasonable to check if the file is actually created in this test.

Reply:
Added a check.

Reply:
I see the test is supposed to create the file, but then there is a check that the file doesn't exist. Looks suspicious.

Reply:
Fixed.